docs/intake-api.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[intake-api]]
+= Intake API
+
+[partintro]
+--
+APM Server exposes a number of endpoints that agents use to deliver data they collect. 
+
+* <<transaction-api>>
+* <<error-api>>
+--
+
+include::./transaction-api.asciidoc[]
+
+include::./error-api.asciidoc[]
```

Comment:
looks like we agree on that ;) https://github.com/elastic/apm-server/pull/549/files#diff-528ff8e4a5598922b19d2e267575d5a8